### ISSUE TITLE
Android device not showing voice output available unless another tts app other than rhvoice is installed. 

### ISSUE THREAD

**Issue by gitbq** [RHVoice/RHVoice#708]
Hi devs, really appreciate all the work in maintaining this super useful app. 

I have tried using calyx os and Graphene os devices running Android 12 and 13.
I am trying to use (1)opentracks - fitness tracker and (2) organic maps and have faced the same challenges in all situations. 

If I install only RHVoice, but when I go into opentracks, instead of voice commands I only get beeps. According to opentracks this is because the TTS software hasn't been configured properly.
When I go to Organic Maps, it says that Voice Instructions are OFF and "not available".

However if I then
1. install another TTS app like espeak
2. Make espeak the default voice text-speech-output app
then under these conditions, voice commands work fine in both apps.
3. Even when I then delete espeak, RH Voice is now recognized and works with these apps. 

Is there any way that RH Voice can be automatically recognised by these apps without first installing a separate TTS app?

**Comment by Olga-Yakovleva** [comment]
When you use RHVoice for the first time with no other TTS installed, 
have any voices been installed in the RHVoice app at that point?

**Comment by gitbq** [comment]
Yes. Initially the CLB voice is installed. The audio could be played the stated that it was already downloaded.

**Comment by pm4rcin** [comment]
I've had that problem too but managed to fix it somehow. I don't remember how it was fixed but I think I have installed espeak, changed to it then change to RHVoice, uninstalled espeak and made RHVoice default one more time.

**Comment by gitbq** [comment]
That's excavation the process I described above. I'm trying to see if the dev is able to make it work without these extra steps.
